algorithmiaio/langpacks
# Algo-701 Scala IPA Language Support

This PR encompasses all langpacks work required to get functioning IPA Scala language support
**To Test:**
* run `./tools/packageset_validator.py -g scala-2 -s scala-2-sbt-1.3.3 -t language -n scala-2`
* Once the langserver responds with `Listening on port 9999`, open a separate terminal and run:
`curl localhost:9999 -H 'Content-Type: application/json' -d '"Algorithmia"'`
* Response should be:  `{"result":"hello Algorithmia","metadata":{"duration":..,"content_type":"text"}}`
** Testing Pre-requisites:**
The following will need to be installed on your system for testing:
* Docker
* Python 3.x, and the pip package `docker`
